C-Plus-Plus/math/check_palindrome.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+     int n, num, digit, rev = 0;
+     cout << "Enter a positive number: ";
+     cin >> num;
+     n = num;//storing original value of num
+     do
+     {
+         digit = num % 10;
+         rev = (rev * 10) + digit;
+         num = num / 10;
+     } while (num != 0);
+
+     if (n == rev)
+         cout << " The number is a palindrome.";
+     else
+         cout << " The number is not a palindrome.";
+    return 0;
+      //sample input 1
```

Comment:
Use multi-line comment instead of using multiple single line comments. The sample I/O should be after ```}```. Refer to other merged PRs for better insight.